### PR TITLE
Branch sync master into dev 1.19

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Sync master with dev-1.19

cc: @zacharysarah @jimangel @kbarnard10

### CODE REVIEW

**Comment by jimangel** [comment]
Going over some of the previous things that bit us:
- [x] No invalid commit message labels
- [x] Merge PR is close to the # of commits that `dev-1.19` is behind (754 vs 763)
- [ ] Verified no commits are bad (red x)

image

However, I also see those commits reflected the exact same on k/website too. I would like to get another set of eyes on this to confirm it's ok to merge.

/lgtm
/cc @zacharysarah @sftim @kbarnard10 @kbhawkey @onlydole

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -189,7 +190,12 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
+<<<<<<< HEAD
     kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
+=======
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname -n=development
+    kubectl scale deployment snowflake --replicas=2 -n=development
+>>>>>>> upstream/master
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:198]: This looks a lot like a not-yet-resolved conflict that got committed in error.

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/namespaces.md:198]: /lgtm cancel

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/namespaces.md:198]: @savitharaghunathan you can:
- edit that to fix it
- `git add content/en/docs/tasks/administer-cluster/namespaces.md `
- `git commit --amend`
- `git push origin:merged-master-dev-1.19 --force-with-lease` (assuming that the remote for your fork is called “origin”)

Comment (kbhawkey) [reply content/en/docs/tasks/administer-cluster/namespaces.md:198]: Good catch @sftim. Just checking, this is a fix to the master branch?

**Comment by kbhawkey** [comment]
Looking, the first red x commit,
pull request 21743 merged to branch, `dev-1.17-ja.2`?

**Comment by kbhawkey** [comment]
I clicked on a few the `red x` commits.
There are a few deployment failures, not sure if these were fixed later.
Here is one log:
https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/5ee666e79b4f7200077dab46

**Comment by sftim** [comment]
AIUI you can get red commits simply because tests never ran for those commits.

File: content/en/docs/tasks/administer-cluster/namespaces.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/namespaces.md:198]: The merge residue is in this branch.

**Comment by kbhawkey** [comment]
Note: https://github.com/kubernetes/website/pull/21746 put pull request 21743 onto master branch.

File: content/en/docs/tasks/administer-cluster/namespaces.md (hunk 1 above)
Comment (savitharaghunathan) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: @sftim I have fixed this and force pushed the branch. Thank you for the commands :)

**Comment by sftim** [comment]
It'd be great to test the result of a simulated merge into the target branch, rather than the to-be-merged branch. I don't think Netlify's GitHub integration lets you do that though.

I previewed locally.
/lgtm

**Comment by jimangel** [comment]
/lgtm
Thanks @savitharaghunathan for your continued leadership and responsiveness.

**Comment by jimangel** [comment]
With one other /lgtm I think this is good to go:
/approve

**Pull request merged**